### ISSUE TITLE
Missing 'DNA binding' annotations for some transcription factors

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2046]
The 2012 transcription annotation guidelines recommended to annotate any transcription factor with 
a)	A term under the DNA binding branch by EXP
b)	A term under the regulation of transcription (BP) by EXP
c)	A term under transcription factor by IC with the a and b annotations. 

While working on https://github.com/geneontology/go-ontology/issues/16152, I found 126 proteins annotated to 'GO:0000981 DNA-binding transcription factor activity, RNA polymerase II-specific' or children, but that didn't have any DNA binding annotation. The list of proteins is in the following document: 

https://docs.google.com/spreadsheets/d/1Pn0p1GHz8nb1Au7eyMoj2Txb4c1EesdyUKp-IaqRdnI/edit#gid=0

Impacted groups: 

Proteins annotated to GO:0000981 or a child, but no DNA binding   annotation | By group | complete?
-- | -- | --
PomBase | 33 | done (not required)
FlyBase | 31
MGI | 23
UniProt | 9
CGD @marekskrzypek | 6
BHF-UCL | 6 | done
RGD | 5
SGD | 4 | done
PINC | 3
ComplexPortal | 1
UniProtKB | 1
AgBase | 1
WB | 1
ZFIN | 1
CACAO @sandyl27  | 1

**Comment by ValWood** [comment]
for example SPAC1142.08 has ISM from   alignments in 
https://www.sciencedirect.com/science/article/pii/S0378111904007929?via%3Dihub#fig1
This is the best evidence for this TF at present, but everyone *knows* its a TF...

I removed from 
SPCC18.03 so I will update the spreadsheet for this one.

**Comment by pgaudet** [comment]
I did not include ISS and descendants, nor IEAs. (I figured that if there was an EXP/IC to GO:0000981 DNA-binding transcription factor activity, RNA polymerase II-specific' or children, there could have been an EXP/IC to the DNA binding term. Maybe that reasoning wasn't correct ?

**Comment by ValWood** [comment]
If we know something is a transcription factor, sometimes the experiments only show transcription but we only have 90 and it's universally accepted among the pombe community that these are all bona fida TFs (these   are the ones which belong to protein families which are exclusively transcription factors

- transcription factor, zf-fungal binuclear cluster type
-  forkhead TFs
-  CBF1/Su(H)/LAG-1 family transcription factor
- Atf-CREB family
- GATA
and some others

We exclude TF annotation from proteins with domains that are  only  *sometimes* associated with TFs like  zf-C2H2 type

Our community need the list of all TF's (a lot), and this is the only way we can provide a comprehensive list.

We have  some DNA-binding TFs with many papers  showing gene specific transcription, microarrays, chromatin association, but no actual DNA binding assay.... I don't think this matters if demonstrated by close family membership or orthology are known to be DNA binding...

and of course we have some that are not published at all but we still want them to be in the TF list...

**Comment by pgaudet** [comment]
Hi @ValWood 

For me it's OK to use these terms even without direct evidence for DNA binding. That data can be captured when it's available. The reason I opened the present ticket is that some people felt they may loose information if the terms are merged as suggested in https://github.com/geneontology/go-ontology/issues/16152. What I understand is that for you it's not the case. Right ? 

Thanks, Pascale

**Comment by pgaudet** [comment]
@bmeldal There is IDA for which activity ? 
(as I said, for me the merge has no impact, but people were concerned that this would lead to loss of information - in these two cases it seems like it dosen't). 

Pascale

**Comment by bmeldal** [comment]
GO:0001077 RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription

At least that's what I curated 2 years ago.

And yes, they both seems to be referring to DNA binding properties - it always confuses me but if terms are in different branches I add them separately. So yes, the merge sounds like a good idea!

**Comment by pgaudet** [comment]
GO:0001077 RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
- refers to the tx branch
- label has been simplified to 'proximal promoter DNA-binding transcription activator activity, RNA polymerase II-specific' 
- this is one of the terms I am planning to merge. So if you have additional evidence for DNA binding, it should be captured as a separate annotation, possibly ''RNA polymerase II proximal promoter sequence-specific DNA binding'' 

Thanks, Pascale

**Comment by ValWood** [comment]
> And yes, they both seems to be referring to DNA binding properties - it always confuses me but if terms are in different branches I add them separately. So yes, the merge sounds like a good idea!

Yes, I have been talking about this a lot with @pgaudet  recently. It leads to lots of inconsistent annotation...We should two different terms to describe   a single activity ( or process).

**Comment by pgaudet** [comment]
@ValWood 
> we should two different terms

or should not ?

**Comment by ValWood** [comment]
shouldn't need...updated comment.

**Comment by bmeldal** [comment]
Thanks, Pascale. I looked at the term but didn't see that the name had been changed - the new one makes more sense as it's the Tx term. We can't pick up name changes (we get the syntax error list from Tony/Aleks so do update do-not-annotate and obsolete terms - if I missed them on here) but that check doesn't check for names.

As for the specific annotation, I had another look as something didn't seem right - and my gut feeling was correct: 

There is Tx evidence (by Luciferase assay) for human and mouse but actual DNA-binding evidence only for mouse (in 2 different papers). AND - the papers don't tell me which part of the regulatory region the TF binds to so I have changed the annotations to their immediate parents: 
GO:0001228 DNA-binding transcription activator activity, RNA polymerase II-specific
and
GO:0000977 RNA polymerase II regulatory region sequence-specific DNA binding

I've also changed both human annotation to ECO:0005544 "complex ISS" from mouse as I can then capture the DNA-binding. And added 
GO:0003713 transcription coactivator activity [IDA] for human.

Thanks for picking up on this inconsistency - it did make me look again! I'm so glad we have our files in order now so your checks pick up our mistakes!
